tests/integration/backward_compatible/authentication_tests/authentication_test.py

```diff
@@ -0,0 +1,60 @@
+import os
+import unittest
+
+from hazelcast.errors import HazelcastError
+from tests.base import HazelcastTestCase
+from tests.util import get_abs_path, set_attr, is_client_version_older_than
+from hazelcast.client import HazelcastClient
+
+try:
+    from hazelcast.security import BasicTokenProvider
+except ImportError:
+    pass
+
+
+@set_attr(enterprise=True)
+@unittest.skipIf(
+    is_client_version_older_than("4.2.2"), "Tests the features added in 4.2.2 version of the client"
```

Comment:
Why `4.2.2`? We are about to release `4.2.1`

Reply:
Don't know why I was thinking we are about to release 4.2.2. Fixed: https://github.com/hazelcast/hazelcast-python-client/pull/446/commits/1e68b2525e4de8cae7891a7fe90ac287d16d0173